### DISCUSSION TITLE
村中人的名与实 | /posts/dididi/

### DISCUSSION THREAD

**Discussion by giscus** [earfanfan/yuanfan#601] [Announcements]
https://yuanfan.rbind.io/posts/dididi/

**Comment by botsp** [comment]
一直很难想象有茶馆的社区生活是怎样的。不管是鲁迅的、还是老舍的，那样的生活都有距离感。

河南人土里刨食，要专门设置一个地方饮茶喝水真是效率不高、效益划不来的事情，倒不如街头支一个面摊卖碗面条，这样于生存才是提高效率的方式、毕竟一碗面条、解渴亦饱腹。这真是思维的不同，华老栓的江南小镇与骆驼祥子的北京都还能把吃和喝分开，中原人民则本能上不这样想。

到现在也进不去咖啡馆，虽然他们做的更好喝、但速溶咖啡仍是俺的最爱，哈哈。

**Reply by earfanfan** [reply]
其实我们村的茶馆只是名义上的称呼，实际上就是大家聚在一起打麻将的棋牌馆。我们家隔壁的茶馆也不是专职只开茶馆的，平时也种田养虾。

吃和喝分开的时候，喝通常也是和其他消遣形式绑定在一起的，比如喝茶看戏听曲子这样的，不然店铺老板没得回本了，确实也如你所说，有钱有闲看戏听曲子的都得是农业社会经济稍微发达的地区才有。

至于喝茶嘛，我记得很小的时候家里有茶叶的话会用热水冲茶水招待客人，但是近几年已经没有这一项了。倒是去年年底我去宜昌那边玩，山下山上的店家招待客人还都是冲有茶叶的茶水的。最近看一位在湖南生活过的老人写的书，受到无形安利很想尝尝湖南的豆子芝麻茶。咖啡我很少喝，前几年还有点喜欢喝芬达和肥宅快乐水，这几年就喜欢喝白水了，或者再泡点红枣片。

**Reply by botsp** [reply]
嗯嗯，是的，在不同地方的风俗习惯里能找到很有意思的差别。
还有可能也是因为河南能产茶叶的地区不多，南部有些地方气候可以，唯一产茶叶有名的信阳，还喜欢把自己划入湖北，哈哈。

麦当劳无糖可乐推荐，喝久了甚至都分不出自己喝的是否含糖了。

**Reply by think-crow** [reply]
**[提示]** 这里撤回了一条评论。

**Reply by earfanfan** [reply]
@think-crow  嗨，其实 botsp 就是河南人，你误会他了。我猜你俩大概都是河南老乡。

看你留言的时间是6小时前，看来昨晚没有好好睡觉，我想起几句歌词，如下：

>一千个噩梦</br>
>换一千个小偷</br>
>够不够 够不够</br>
>偷走你的难过

今天是2025年的第一天，新年好，希望你今夜有个好梦。

**Reply by think-crow** [reply]
谢谢！
